### PR TITLE
Add release script

### PR DESCRIPTION
**Pull request by matejcik**
couple coin-side changes:
- Wanchain now has a proper eth entry and chain id (https://github.com/wanchain/go-wanchain/releases/tag/v2.1.10-beta.1) so I removed it from misc.
  * this could use a code change, as we're using magic to detect wanchain, but AFAICT it will continue to work
- Ontology also has an eth entry! they have two tokens, ONT (custom mainnet) and ONG (Ethereum chain). We now support the latter. This could cause confusion.
- I removed `misc:ONT` entry from the repository as it was there only as a placeholder and proper ONT implementation doesn't seem to be happening anytime soon

There's now a release script. `common/tools/release.sh` will:
1. take a snapshot of `cointool.py check`
2. update `chains` and `tokens` submodules
3. run `support.py release` which looks at everything that is NOT marked supported/unsupported, adds all tokens, and asks about each and every chain (those must be reviewed manually for now)
4. diff against current result of `cointool.py check` so you see if you broke something
5. regenerates `coins_details.json`

(`coins_details` will need to be regenerated later anyway because they list all new additions as "soon", because they're not in a released firmware yet. so we might be able to remove it from `release.sh`)

the PR also includes results of the latest run of `release.sh`

### CODE REVIEW

**Comment by matejcik** [comment]
failure of changelog is spurious, only generated files were changed.

i thought there was an ignore-list for generated files, but can't find it now?

**Review by tsusanka — APPROVED** [review]
LGTM. One nit.

File: common/tools/release.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+fi
+
+CHECK_OUTPUT=$(mktemp -d)
+trap "rm -r $CHECK_OUTPUT" EXIT
+
+$HERE/cointool.py check > $CHECK_OUTPUT/pre.txt
+
+ETH_DIR=$HERE/../defs/ethereum
+
+for dir in chains tokens; do
```
[the remark this answers is not in the document]
Comment (matejcik) [reply common/tools/release.sh]: fixed in 63602c8

**Comment by matejcik** [comment]
in 63602c8 i also remove `coins_details`, because, as discussed, "at time of release" is not a good time to regenerate anyway

**Review by tsusanka — APPROVED** [review]

**Pull request merged**